src/Peachpie.CodeAnalysis/Semantics/PhpOperationVisitor.cs

```diff
@@ -130,7 +130,11 @@ public virtual void VisitList(BoundListEx x)
 
         public virtual void VisitFieldRef(BoundFieldRef x)
         {
-            VisitTypeRef(x.ParentType);
+            if (x.ParentType != null)
```

Comment:
Just an unnecessary piece code.
The method handles `null` already for the reason so any call to `VisitTypeRef` can be less complex, less of conditional code, less less braces etc. ... otherwise you could copy/paste `if (blahblah != null) { VisitTypeRef(blahblah); }` for 16 times and maintain it to the rest of our lives